README.md

```diff
@@ -121,17 +132,17 @@ To enable a debug build, set the CMAKE_BUILD_TYPE flag (default is `Release`):
 
 ```bash
 # cmake -DCMAKE_BUILD_TYPE=Debug -DCOVERAGE=ON ..
-# make coverage
+# cmake --build build --target coverage
 ```
 
 Other targets:
 
 ```
-$ make pretty        # Run clang-format on all source files
-$ make lint          # Run cppcheck
-$ make coverage      # Just coverage.info for codecov.io
-$ make coverage-html # Local HTML report
-$ make coverage-data # Cobertura XML report
+$ cmake --build build --target pretty        # Run clang-format on all source files
+$ cmake --build build --target lint          # Run cppcheck
+$ cmake --build build --target coverage      # Just coverage.info for codecov.io
+$ cmake --build build --target coverage-html # Local HTML report
+$ cmake --build build --target coverage-data # Cobertura XML report
```

Comment:
Why aren't we using make for this. It seemed easier before

Reply:
hi @sphinxc0re long-time-lurker, feel free to ignore my drive-by comment. I like to run cmake using the ninja generator as it generally leads to faster builds. Having generator-agnostic docs is always a pleasant surprise.

Side note: building master from clean clone fails on first `make -j9` run with ```
.../das-labor/neopg/build/include/neopg-tool/dump_packet_sink.h:8:10: fatal error: neopg/openpgp.h: No such file or directory
 #include <neopg/openpgp.h>``` yet the ninja build is no probelm. This seems like a cmake error though.

Reply:
@mmlb This feels like you should open an issue.

Reply:
> This seems like a cmake error though.
>
It's more a combination of the project structure and the known problem of parallel builds with `make`. That's why for example, the openssl build system prevent parallel builds with make :-).